src/app/components/login/login.component.ts

```diff
@@ -31,17 +31,19 @@ export class LoginComponent {
   }
 
   login() {
-     console.log(this.loginForm.value)
+     console.log( ' Login Form Values:',this.loginForm.value)
+
     if (this.loginForm.valid) {
       const loginRequest: LoginRequest = {
         email: this.loginForm.get('email')?.value,
         password: this.loginForm .get('password')?.value
        }
        this.authService.login(loginRequest).subscribe({
         next: (res: any) => {
-          console.log(res)
+          console.log('Login Success:' ,res)
           this.authService.setLoggedIn(true)
           this.router.navigate(['add-tech'])
```

Comment:
use navigateByUrl since you're only navigating by the url noother parameters attached

Reply:
done

Reply:
Done now using navigateByUrl